### DISCUSSION TITLE
How I can add some functions to this project?

### DISCUSSION THREAD

**Discussion by ghostersk** [ChrisTitusTech/winutil#685] [Q&A]
Hi I would like to add some additional functions to this project however I am not sure what files i need edit/add where?
I want to add couple functions, one is enable F8 key and second is to re-enable Registry Key Backups
It just things from these video https://www.youtube.com/watch?v=eY76UVXxENo&t=901s

Notes from video:
Registry:
HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Session Manager\Configuration Manager

Add Dword 32Bit [EnablePeriodicBackup] set to 1
Add Dword 32Bit [BackupCount] Set to desired number of backups  <<< This would be good to be able set dynamicaly I have 4

Command For Task scheduler <<<< This task would need option to set how often this should run
schtasks /run /i /tn "\Microsoft\Windows\Registry\RegIdleBackup"

Enable F8 menu
New Key [LastKnownGood]
New Dword 32Bit [Enabled] set to 1

In an administrator CMD ( my note: not sure if Current or Default is needed)

bcdedit /set "{Current}" BootMenuPolicy Legacy

bcdedit /set "{default}" bootmenupolicy legacy

**Comment by DeveloperDurp** [comment]
Sounds like that would be adding the values into the tweaks.json file with your values and then adding a checkbox in the gui. The program will match the name of the checkbox with what's inside the json file. 

https://github.com/ChrisTitusTech/winutil/blob/main/config/tweaks.json
https://github.com/ChrisTitusTech/winutil/blob/main/xaml/inputXML.xaml
